src/plugins/condition/StyleRuleManager.js

```diff
@@ -74,26 +85,33 @@ export default class StyleRuleManager extends EventEmitter {
             if (foundStyle !== this.currentStyle) {
                 this.currentStyle = foundStyle;
             }
+            this.updateDomainObjectStyle();
         } else {
-            if (this.currentStyle !== this.defaultStyle) {
-                this.currentStyle = this.defaultStyle;
-            }
+            this.applyStaticStyle();
         }
-
-        this.updateDomainObjectStyle();
     }
 
     updateDomainObjectStyle() {
-        this.emit('conditionalStyleUpdated', this.currentStyle)
+        if (this.callback) {
+            this.callback(Object.assign({}, this.currentStyle));
+        }
     }
 
-    destroy() {
-        if (this.currentStyle) {
-            Object.keys(this.currentStyle).forEach(key => {
-                this.currentStyle[key] = 'inherit';
-            });
+    applyStaticStyle() {
+        if (this.staticStyle) {
+            this.currentStyle = this.staticStyle.style;
+        } else {
+            if (this.currentStyle) {
+                Object.keys(this.currentStyle).forEach(key => {
+                    this.currentStyle[key] = 'transparent';
```

Comment:
Looks like this is used later on to identify which styles should be deleted from the element's style definition. If so, can we use something other than `transparent`? This is a valid CSS value for some style rules, so it may not be clear to developers that this will result in the subsequent deletion of this style rule.  Could use a distinct value save in a named constant instead?

Reply:
Yes. Will do.